src/components/gallery/GalleryItem.tsx

```
@@ -0,0 +1,50 @@
+        <img src={template.thumbnail} alt={template.title} draggable={false} />
+      </div>
+      {/* hover 했을떄 나타나기 */}
+      <div
+        className={cn(
+          "absolute left-0 top-0 h-full w-full bg-black/70 p-4 transition-all duration-300 ",
+          "flex items-end justify-end",
+          "hidden group-hover:flex"
+        )}
+      >
+        <button className="flex cursor-pointer items-center gap-2 text-base text-white hover:underline">
+          <ArrowRightIcon className="text-white" size={18} />
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)